cmd/cloud/user.go

```diff
@@ -0,0 +1,62 @@
+package cloud
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+var role string
+
+func newUserCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "user",
+		Aliases: []string{"us"},
+		Short:   "Invite Users to your Astro Organization",
+		Long:    "Invite Users to your Astro Organization.",
+	}
+	cmd.SetOut(out)
+	cmd.AddCommand(
+		newUserInviteCmd(out),
+	)
+	return cmd
+}
+
+func newUserInviteCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "invite",
+		Aliases: []string{"inv"},
+		Short:   "Invite a user to your Astro Organization",
+		Long:    "astro user invite [email] --role [ORGANIZATION_MEMBER, ORGANIZATION_BILLING_ADMIN, ORGANIZATION_OWNER].",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return userInvite(cmd, args, out)
+		},
+	}
+	// TODO Check if ORGANIZATION_MEMBER is the correct default?
+	// TODO Check if the 3 roles below are the only ones allowed?
```

Comment:
These are the three organization roles we have for now. These may change in the future